bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/LabelImpl.java

```diff
@@ -78,6 +78,10 @@ public Boolean isVisible() {
      */
     @Override
     public String getValue() {
-        return ComponentUtils.translate(properties.get(PN_TITLE, this.defaultTitle), PN_TITLE, properties, i18n);
+        if (properties.containsKey(PN_TITLE))
+            return ComponentUtils.translate(properties.get(PN_TITLE, this.defaultTitle), PN_TITLE, properties, i18n);
+        else
+            return ComponentUtils.translate(properties.get(PN_TITLE, ""), PN_TITLE, properties, i18n);
```

Comment:
Please see if we are saving "" in AF v1 or "\n".
Also check if the same is being reflected on content tree as well?